.gitignore

```diff
@@ -29,4 +29,4 @@ test/server-test-config/ssl-cert.pem
 test/server-test-config/ssl-key.pem
 test/server-test-config/plugin-config-data/
 
-docs/dist
+docs/built
```

Comment:
This should be `docs/dist`. I used to be `docs/built`, but I changed it in #1914.